tests/command.bats

```diff
@@ -211,6 +292,27 @@ load '/usr/local/lib/bats/load.bash'
   unset BUILDKITE_PLUGIN_ARTIFACTS_DOWNLOAD_TO
 }
 
+@test "Post-command uploads artifacts with a compressing option" {
+  mkdir /tmp/data && touch /tmp/data/foo.log
```

Comment:
some of these tests are failing - i think you'll need to clean up your temp directories after each test as they're conflicting with each other.

Reply:
@chrisfowles I have fixed it with `mkdir `option ignore error if existing.
I can run test successfully.

Reply:
We should still clean up after ourselves to ensure that we're not impacting any other downstream tests in the future

Reply:
@chrisfowles ok Chris, help me squash commit  before merged to master.